matt-Dragoris/hw_4/MonkeyObj.js

```diff
@@ -0,0 +1,42 @@
+function monkey(name, species) {
+    this.name = name,
+    this.species = species,
+    this.foodsEaten = [],
+    this.hangry = false
+}
+
+monkey.prototype.eatSomething = function(food){
+    if(this.hangry) {
+        this.hangry = false;
+        console.log (`Thanks for the ${food}, I needed that. *${this.name} is now your friend!*`)
+    }
+    this.foodsEaten.push(food);
+}
+
+monkey.prototype.introduce = function() {
+    if(this.foodsEaten.length) {
+        console.log(`Hi! My name is ${this.name} the friendly ${this.species}, and ive eaten some ${this.foodsEaten.join(', ')}`);
+    }else {
+        this.hangry = true;
+        console.log(`Me ${this.name} the enraged ${this.species}, i demand food! FEED ME!!`);
```

Comment:
Feeding loco the ape was an absolute rollercoaster, haha!

    Me loco the enraged ape, i demand food! FEED ME!!
    Thanks for the grass, I needed that. *loco is now your friend!*
    Hi! My name is loco the friendly ape, and ive eaten some grass, berries